sv/SDVTrafficState.py

```diff
@@ -298,62 +363,123 @@ def get_traffic_state(
         road_occupancy = road_occupancy
     )
 
-def fill_occupancy(my_vehicle:Vehicle, lane_config:LaneConfig, traffic_vehicles, traffic_vehicles_orp, lanelet_map:LaneletMap, intersections):
+def fill_occupancy(my_vehicle:Vehicle, lane_config:LaneConfig, traffic_vehicles, traffic_vehicles_orp, lanelet_map:LaneletMap, intersections, euclid_only=False, frenet_only=False):
     '''
         Identify vehicles in strategic zones using the (Frénet Frame) and assign their id.
         Road Occupancy contains only one vehicle per zone (closest to SDV)
     '''
-    half_length = my_vehicle.length/2
+    three_quarter_length = my_vehicle.length / 1.5
```

Comment:
that's not correct math. lenght /1.5 is two thirds. Why not simply `length * 0.75`? It's evident that's "three quarters".